package.json

```diff
@@ -157,7 +157,6 @@
         "windows": {
           "program": "./coreclr-debug/debugAdapters/OpenDebugAD7.exe"
         },
-        "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
```

Comment:
OK -- will add that back. Sorry about that! I wasn't sure because the debugger code just dug it out of package.json.